docs/content/usage/configuration/credentials.md

```diff
@@ -5,6 +5,13 @@ draft: false
 description: "Configure credentials that the Infection Monkey will use for propagation."
 ---
 
-On this screen you can feed the Infection Monkey "stolen" credentials from your network, simulating an attacker with inside knowledge.
+On this screen, you can configure credentials for Infection Monkey to use in the propagation
+stage. These can be anything from default or weak passwords, to "stolen" credentials from your
+network, simulating an attacker with inside knowledge.
+
+All possible credential combinations will be tried for propagation. Identity-secret pairs added
+in the same row will be attempted before any other credential combinations.
+
+Note that to add a credential, it is not necessary to fill out all the fields.
```

Comment:
I don't think we need this note

Reply:
Agreed, an example would be better.

Reply:
The screenshot shows this. Do you think we need something else?

Reply:
I think we can just get rid of this line. The screenshots clearly show that you don't need to fill out all fields in a row.